app/controllers/searches_controller.rb

```diff
@@ -232,7 +232,7 @@ def clusters_belong_to
   def close_friends
     @user_items = build_user_items(@searched_tw_user.close_friends)
     @graph = @searched_tw_user.close_friends_graph
-    @tweet_text = view_context.close_friends_text(@user_items.slice(0, 3).map{|i| i[:target] }, @searched_tw_user)
+    @tweet_text = view_context.close_friends_text(@user_items.slice(0, 3).map { |i| i[:target] }, @searched_tw_user)
```

Comment:
__[rubocop]__
* Line is too long. [86/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)